distributed/comm/ucx.py

```diff
@@ -107,6 +109,37 @@ def device_array(n):
             pool_allocator=True, managed_memory=False, initial_pool_size=pool_size
         )
 
+def get_create_endpoint():
+    global reuse_endpoints_create
```

Comment:
Should we consider using [the Dask config]( https://docs.dask.org/en/latest/configuration.html ) for this?

Reply:
Not sure I understand your question here @jakirkham , it's already using Dask config below in line 121. Here the global is just to keep track of how it's reusing (or not) the endpoints.

Reply:
I missed that initially. Thanks for pointing that out.